ios/StatusIm.xcodeproj/project.pbxproj

```diff
@@ -451,7 +451,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -o errexit\nexport NODE_BINARY=\"node\"\nexport NODE_ARGS=\" --max-old-space-size=16384 \"\n\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "set -o errexit\nexport NODE_BINARY=\"node\"\nexport NODE_ARGS=\" --max-old-space-size=16384 \"\n\nbash -x ../node_modules/react-native/scripts/react-native-xcode.sh 2>&1 > /tmp/xcode.log\n";
```

Comment:
We could possibly leave it here, but we can drop the `bash -x` part since it does nothing, and maybe redirect logs to current directly instead of `/tmp` with a more descriptive name, since it's clearly useful for debugging.